pkg/kubectl/history_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes/fake"
+	"testing"
+)
+
+func Test_HistoryViewerFor(t *testing.T) {
+	fakeClientset := &fake.Clientset{}
+	daemonSetKind := schema.GroupKind{Group: "apps", Kind: "DaemonSet"}
+	result, err := HistoryViewerFor(daemonSetKind, fakeClientset)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if _, ok := result.(*DaemonSetHistoryViewer); !ok {
+		t.Fatalf("unexpected output type")
```

Comment:
Give each test case a name, so it will be easier to locate which one fails.

Same comments for all test cases.